lib/generators/sufia/work/templates/locale.en.yml.erb

```diff
@@ -1,4 +1,10 @@
 en:
+  sufia:
+    select_type:
+      <%= file_name %>:
+        name:               "<%= human_name.titleize %>"
+        description:        "<%= human_name %> works"
```

Comment:
@rotated8 is this sufficient? https://github.com/projecthydra/sufia/wiki/Customizing-Metadata